### PR TITLE
Fix NPE

### LINKED ISSUE #4320: java.lang.NullPointerException: Cannot invoke "String.contains(java.lang.CharSequence)" because "rawSDesc" is null
**Describe the bug**
Getting a null pointer exception

**To Reproduce**
I'm not sure. This usually happens on the AI opponent's turn after clearing the stack. 

**Expected behavior**
No NullPointerException

**Screenshots**
https://imgur.com/SWUToMx

**Desktop (please complete the following information):**
 - Windows
 - Chrome
 - 1.6.59
 
**Additional context**
```
java.lang.NullPointerException: Cannot invoke "String.contains(java.lang.CharSequence)" because "rawSDesc" is null
	at forge.game.ability.SpellAbilityEffect.getStackDescriptionWithSubs(SpellAbilityEffect.java:81)
	at forge.game.spellability.AbilitySub.getStackDescription(AbilitySub.java:100)
	at forge.game.ability.SpellAbilityEffect.getStackDescriptionWithSubs(SpellAbilityEffect.java:128)
	at forge.game.spellability.AbilitySub.getStackDescription(AbilitySub.java:100)
	at forge.game.trigger.Trigger.replaceAbilityText(Trigger.java:186)
	at forge.game.trigger.WrappedAbility.getStackDescription(WrappedAbility.java:237)
	at forge.game.spellability.SpellAbilityStackInstance.<init>(SpellAbilityStackInstance.java:71)
	at forge.game.zone.MagicStack.push(MagicStack.java:461)
	at forge.game.zone.MagicStack.add(MagicStack.java:334)
	at forge.game.zone.MagicStack.add(MagicStack.java:221)
	at forge.ai.ComputerUtil.playStack(ComputerUtil.java:271)
	at forge.ai.PlayerControllerAi.orderAndPlaySimultaneousSa(PlayerControllerAi.java:1086)
	at forge.game.zone.MagicStack.chooseOrderOfSimultaneousStackEntry(MagicStack.java:830)
	at forge.game.zone.MagicStack.addAllTriggeredAbilitiesToStack(MagicStack.java:790)
	at forge.game.phase.PhaseHandler.checkStateBasedEffects(PhaseHandler.java:1161)
	at forge.game.phase.PhaseHandler.startFirstTurn(PhaseHandler.java:1047)
	at forge.game.GameAction.startGame(GameAction.java:2114)
	at forge.game.Match.startGame(Match.java:90)
	at forge.gamemodes.match.HostedMatch$2.run(HostedMatch.java:255)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

### PR DESCRIPTION
**Pull request by tool4ever**
Closes #4320

### CODE REVIEW

**Review by Agetian — APPROVED** [review]

File: forge-gui/res/cardsfolder/c/chancellor_of_the_forge.txt [hunk 1]
```diff
@@ -8,8 +8,7 @@ SVar:X:Count$Valid Creature.YouCtrl
...
 SVar:TokenOnUpkeep:DB$ Effect | Triggers$ TrigToken | Name$ Chancellor of the Forge effect | SpellDescription$ You may reveal this card from your opening hand. If you do, at the beginning of the first upkeep, create a 1/1 red Phyrexian Goblin creature token with haste.
-SVar:TrigToken:Mode$ Phase | Phase$ Upkeep | Execute$ EffToken | TriggerDescription$ At the beginning of the first upkeep, create a 1/1 red Phyrexian Goblin creature token with haste.
-SVar:EffToken:DB$ Token | TokenAmount$ 1 | TokenOwner$ You | TokenScript$ r_1_1_phyrexian_goblin_haste | SubAbility$ RemoveMe
-SVar:RemoveMe:DB$ ChangeZone | Defined$ Self | Origin$ Command | Destination$ Exile
+SVar:TrigToken:Mode$ Phase | Phase$ Upkeep | Execute$ EffToken | OneOff$ True | TriggerDescription$ At the beginning of the first upkeep, create a 1/1 red Phyrexian Goblin creature token with haste.
+SVar:EffToken:DB$ Token | TokenAmount$ 1 | TokenOwner$ You | TokenScript$ r_1_1_phyrexian_goblin_haste
 SVar:PlayMain1:TRUE
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/c/chancellor_of_the_forge.txt:13]: Probably a `DeckHas:Type$Goblin` too?

File: forge-gui/res/cardsfolder/c/chancellor_of_the_spires.txt [hunk 2]
```diff
@@ -6,9 +6,8 @@ K:MayEffectFromOpeningHand:RevealCard
...
-SVar:Mill:DB$ Mill | NumCards$ 7 | Defined$ Player.Opponent | SubAbility$ RemoveMe
-SVar:RemoveMe:DB$ ChangeZone | Defined$ Self | Origin$ Command | Destination$ Exile
+SVar:TrigMill:Mode$ Phase | Phase$ Upkeep | Execute$ Mill | OneOff$ True | TriggerDescription$ At the beginning of the first upkeep, each opponent mills seven cards.
+SVar:Mill:DB$ Mill | NumCards$ 7 | Defined$ Player.Opponent
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerDescription$ When CARDNAME enters the battlefield, you may cast target instant or sorcery card from an opponent's graveyard without paying its mana cost.
 SVar:TrigChangeZone:DB$ Play | TgtZone$ Graveyard | WithoutManaCost$ True | ValidTgts$ Instant.OppOwn,Sorcery.OppOwn | TgtPrompt$ Choose target instant or sorcery card from an opponent's graveyard | Optional$ True
 Oracle:You may reveal this card from your opening hand. If you do, at the beginning of the first upkeep, each opponent mills seven cards.\nFlying\nWhen Chancellor of the Spires enters the battlefield, you may cast target instant or sorcery card from an opponent's graveyard without paying its mana cost.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/c/chancellor_of_the_spires.txt:13]: DeckHas:Ability$Mill ?

File: forge-gui/res/cardsfolder/c/chancellor_of_the_forge.txt [hunk 3]
```diff
@@ -7,9 +7,9 @@ SVar:EffMassToken:DB$ Token | TokenAmount$ X | TokenOwner$ You | TokenScript$ r_
...
 K:MayEffectFromOpeningHand:RevealCard
 SVar:RevealCard:DB$ Reveal | RevealDefined$ Self | SubAbility$ TokenOnUpkeep | SpellDescription$ You may reveal this card from your opening hand. If you do, at the beginning of the first upkeep, create a 1/1 red Phyrexian Goblin creature token with haste.
-SVar:TokenOnUpkeep:DB$ Effect | Triggers$ TrigToken | Name$ Chancellor of the Forge effect | SpellDescription$ You may reveal this card from your opening hand. If you do, at the beginning of the first upkeep, create a 1/1 red Phyrexian Goblin creature token with haste.
-SVar:TrigToken:Mode$ Phase | Phase$ Upkeep | Execute$ EffToken | TriggerDescription$ At the beginning of the first upkeep, create a 1/1 red Phyrexian Goblin creature token with haste.
-SVar:EffToken:DB$ Token | TokenAmount$ 1 | TokenOwner$ You | TokenScript$ r_1_1_phyrexian_goblin_haste | SubAbility$ RemoveMe
-SVar:RemoveMe:DB$ ChangeZone | Defined$ Self | Origin$ Command | Destination$ Exile
+SVar:TokenOnUpkeep:DB$ Effect | Triggers$ TrigToken | SpellDescription$ You may reveal this card from your opening hand. If you do, at the beginning of the first upkeep, create a 1/1 red Phyrexian Goblin creature token with haste.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/c/chancellor_of_the_forge.txt:10]: Should this SpellDescription$ be trimmed too?

**Review by Northmoc — COMMENTED** [review]
👍 thanks for tidying up some of my boo-boos

**Review by Northmoc — APPROVED** [review]

**Pull request merged**